### PR TITLE
Add do not edit warning to generated files

### PR DESCRIPTION
**Pull request by adambanker**

### CODE REVIEW

File: make.paws/R/make_package.R [hunk 1]
```diff
@@ -98,7 +101,9 @@ copy_customizations <- function(api, path) {
   if (length(src_file) > 0) {
     package <- package_name(api)
     filename <- sprintf("%s_customizations.R", package)
-    file.copy(from = src_file, to = file.path(path, filename))
+    warning_text <- unlist(edit_warning(list(), new_line = T))
+    cat(warning_text, file = file.path(path, filename))
```
Comment (davidkretch) [inline make.paws/R/make_package.R]: Can you change this to `write` or `writeChar` instead of `cat`? I feel one of these would be clearer.

**Comment by davidkretch** [comment]
A couple requests:
1. Can you change `edit_warning` to `add_edit_warning`?
2. Can you change the text to "This file is generated by make.paws. Please do not edit here."
3. See other comment.

File: make.paws/R/make_package.R (hunk 1 above)
Comment (adambanker) [reply make.paws/R/make_package.R]: I changed it to write

**Comment by davidkretch** [comment]
Looks good to me.

**Pull request merged**